packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/RecordHistoryEvents/EventRow.js

```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Grid } from 'semantic-ui-react'
+
+import { getYMDHMSFromTimestamp } from 'common/es5/date'
+
+const propTypes = {
+  actionType: PropTypes.string.isRequired,
+  timestamp: PropTypes.string.isRequired,
+  userName: PropTypes.string,
+}
+
+const defaultProps = {
+  userName: '',
+}
+
+class EventRow extends PureComponent {
+  render() {
+    const { actionType, timestamp, userName } = this.props
+
+    const eventTimestamp = getYMDHMSFromTimestamp(timestamp)
+    return (
+      <Grid.Column width={10}>
+        {actionType} {userName} {eventTimestamp}
```

Comment:
if there is no username there will be an extra space here. you should fix so that another space is added only if there is a username